ddl/ddl.go

```diff
@@ -201,28 +198,29 @@ type DDL interface {
 
 // ddl represents the statements which are used to define the database structure or schema.
 type ddl struct {
-	m sync.RWMutex
+	m          sync.RWMutex
+	infoHandle *infoschema.Handle
+	quitCh     chan struct{}
 
-	infoHandle   *infoschema.Handle
-	hook         Callback
-	hookMu       sync.RWMutex
+	*ddlCtx
+	workers []*worker
+}
+
+type ddlCtx struct {
```

Comment:
What is the different between `ddl` and `ddlCtx`?

Reply:
`ddlCtx` will be used in `worker`.

Reply:
The name is not self-explained. Need some comments.